examples/triton-docker-combined/variables.tf

```diff
@@ -0,0 +1,14 @@
+variable "triton_account" {}
+variable "triton_url" {
+  default = "https://us-sw-1.api.joyentcloud.com"
+}
+variable "triton_key_path" {
+  default = "path to your ssh key for triton"
+}
+variable "triton_key_id" {}
```

Comment:
We might as well go all in on that one. I'll make the default something like "run ssh-keygen ..."